app/models/poll/question/answer.rb

```diff
@@ -14,4 +14,18 @@ class Poll::Question::Answer < ActiveRecord::Base
   def description
     super.try :html_safe
   end
+
+  def self.order_answers(ordered_array)
+    ordered_array.each_with_index do |answer_id, order|
+      answer = find(answer_id)
+      answer.update_attribute(:given_order, (order + 1))
+      answer.save
```

Comment:
[`update_attribute`](https://apidock.com/rails/ActiveRecord/Persistence/update_attribute) saves the record already. I would refactor this 3 lines into just `find(answer_id).update_attribute(:given_order, (order + 1))`